website/playground/src/external/react-splitpane.css

```diff
@@ -9,12 +9,10 @@
   -webkit-background-clip: padding;
   background-clip: padding-box;
 }
-
```

Comment:
@g-plane this feels like a bug?

Reply:
Oh, I see that this occurred because the file happened to be UTF-8 w/ BOM (dprint used to maintain the BOM, but now it passes it to the plugins as-is). I manually added back the line breaks.